cg/models/invoice/invoice.py

```diff
@@ -0,0 +1,56 @@
+from pydantic import BaseModel
+from typing import List, Optional, Any
+from cg.constants.priority import PriorityTerms
+from cg.constants.invoice import CostCenters
+from cg.constants.sequencing import RecordType
+
+"""Module for modelling Invoices."""
+
+
+class Contact(BaseModel):
```

Comment:
Should we name them `InvoiceContact, InvoiceApplication` to anchor these to invoive since these terms are also used elsewhere in other contexts?

Reply:
Makes sense!